ocs_ci/ocs/resources/storage_cluster.py

```diff
@@ -623,6 +628,49 @@ def osd_encryption_verification():
             )
             raise ValueError("OSD is not encrypted")
 
+    if ocs_version > version.VERSION_4_6:
+        log.info("Verify luks header label for encrypted devices")
+        worker_nodes = get_worker_nodes()
+        logging.info("NODES: {}".format(worker_nodes))
+        failures = 0
+        failure_message = ""
+        node_obj = OCP(kind="node")
+
+        for n in worker_nodes:
+            luks_devices = node_obj.exec_oc_debug_cmd(
+                node=n,
+                cmd_list=[
+                    "lsblk -o NAME,TYPE,FSTYPE | grep -E 'disk.*crypto_LUKS' | awk '{print $1}'"
+                ],
+            ).split("\n")
```

Comment:
Please create a new function and move it to some common lib.  This looks to be a useful one and can help in other tests